config/singleton_enclave_config.toml

```diff
@@ -78,6 +78,7 @@ enclave_type = "SWE"
 [WorkerConfig]
 # Id of worker in plain text
 worker_id = "singleton-worker-1"
+workloads = "echo-result,heart-disease-eval,inside-out-eval,simple-wallet"
```

Comment:
Please add a comment. "Supported workloads for this worker (this can be overridden by by passing values in command line for enclave manager)"

Reply:
Please add one more comment in workload tutorial README.md, Singleton and in WPE config.toml.  "The work orders matching these workload ids will be processed by the enclave manager. The enclaves should be built with matching workloads".